_redirects

```diff
@@ -0,0 +1 @@
+/learn/platform-overview /learn/platform-overview/README
```

Comment:
@dhrubabasu https://b97b68af.avalanche-docs.pages.dev/learn/platform-overview does not redirect. What did I do wrong?

Reply:
```suggestion
/learn/platform-overview /learn/platform-overview/README 301
```

Reply:
I think this is what you need to do ^